components/dashboard/src/projects/NewProject.tsx

```diff
@@ -348,49 +333,42 @@ export default function NewProject() {
                         </div>
                     </div>)}
                 </div>
 
             </div>
             {isGitHub() && (
-                    <div className="pt-3">
-                        <div className="text-gray-500 text-center w-96 mx-8">
-                            Repository not found? <a href="javascript:void(0)" onClick={e => reconfigure()} className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Reconfigure</a>
-                        </div>
-                        {isGitHub() && noOrgs && (
-                            <div className="text-gray-500 mx-auto text-center">
-                                Missing organizations? <a href="javascript:void(0)" onClick={e => grantReadOrgPermissions()} className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Grant permissions</a>
-                            </div>
-                        )}
+                <div>
+                    <div className="text-gray-500 text-center w-96 mx-8">
+                        Repository not found? <a href="javascript:void(0)" onClick={e => reconfigure()} className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Reconfigure</a>
                     </div>
+                </div>
             )}
-            <p className="text-left w-full mt-12 text-gray-500">
-                <strong>Teams &amp; Projects</strong> are currently in Beta. <a href="https://github.com/gitpod-io/gitpod/issues/5095" target="gitpod-feedback-issue" rel="noopener" className="gp-link">Send feedback</a> or open a <a href="/workspaces" className="gp-link">New Workspace</a> with an example repository.
+            <p className="text-center w-full mt-12 text-gray-500">
+                <strong>Teams &amp; Projects</strong> are currently in Beta. <a href="https://github.com/gitpod-io/gitpod/issues/5095" target="gitpod-feedback-issue" rel="noopener" className="gp-link">Send feedback</a>
             </p>
         </>
         );
 
-        const renderLoadingState = () => (<div>
-            <div className="mt-8 border rounded-xl border-gray-100 dark:border-gray-700 flex-col">
-                <div>
-                    <div className="px-12 py-16 text-center text-gray-500 bg-gray-50 dark:bg-gray-800 rounded-xl w-96 h-h96 flex items-center justify-center">
-                        <h3 className="mb-2 text-gray-400 dark:text-gray-600 animate-pulse">
-                            Loading ...
-                        </h3>
+        const renderLoadingState = () => (
+            <div>
+                <p className="text-gray-500 text-center text-base">Projects allow you to manage prebuilds and workspaces for your repository. <a href="https://www.gitpod.io/docs/teams-and-projects" rel="noopener" className="gp-link">Learn more</a></p>
```

Comment:
**Praise**: Many thanks for adding back this text in the loading screen! (This is actually something I broke recently 🙈)

**Thought**: Maybe in this case, the slight duplication is not a problem, but it still feels a little awkward that someone could possibly fix or improve the copy here (or update the link, etc) but accidentally forget to update the other occurrence of the same text (or vice versa). Maybe it's a tiny bit safer to de-duplicate this text again?

Possible solutions could be:
- `const ProjectsDescription = <div>...</div>;`
    - defined somewhere above `renderLoadingState` and `renderRepos`, in the same scope
    - used as `{ProjectsDescription}`
- `const renderProjectsDescription = () => <div>...</div>;`
    - defined somewhere above `renderLoadingState` and `renderRepos`, in the same scope
    - used as `{renderProjectsDescription()}`
- `function ProjectDescription() { return <div>...</div>; }`
    - defined as a separate component, e.g. at the bottom of the file
    - used as `<ProjectDescription/>`

I'm not really sure which one is cleaner / more readable 😅 please feel free to pick the option you prefer, or to ignore this comment entirely for now.

Reply:
You're totally right and I actually extracted the logic in the beginning. But TypeScript didn't like me doing that 🥲 as it always considered one of the variables undefined even though I checked. I will try extracting the HTML then and maybe that would work. Completely agree that having this duplication is prone to being overlooked and bugs.